tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"compilerOptions": {
 		"strict": false,
-		"target": "es5",
+		"target": "es2015",
```

Comment:
Bumping from `es5` to `es2015` was required since we use the spread operator on iterators; it does not impact the code generated by Next.js.